test-runner/src/cli.ts

```diff
@@ -40,6 +41,8 @@ program
   .option('--output <outputDir>', 'Folder for output artifacts, default: test-results', path.join(process.cwd(), 'test-results'))
   .option('--timeout <timeout>', 'Specify test timeout threshold (in milliseconds), default: 10000', '10000')
   .option('-u, --update-snapshots', 'Use this flag to re-record every snapshot that fails during this test run')
+  .option('--test-match <pattern>', 'Pattern used to find test files', '**/?(*.)+(spec|test).[jt]s')
```

Comment:
How does this relate to `command.args.slice(1)` ? I.e. when do I use one vs the other?

Reply:
Aside from the difference in parsing the patterns, test match is ANDed with filters. See https://jestjs.io/docs/en/configuration#testmatch-arraystring and https://jestjs.io/docs/en/cli#jest-regexfortestfiles.

Reply:
You use a filter "mouse" when you want to run "mouse.spec.ts" not "keyboard.spec.ts". You use --test-match when you want to run "mouse.spec.ts" not "mouse.ts".

Reply:
The second part seems confusing, why can't I just say `npm run test mouse.ts`?